tests/unit/executor/test_part_handler.py

```diff
@@ -14,6 +14,9 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+# pylint: disable=too-many-lines
```

Comment:
This was the only warning I disabled.  Reformatting made the file 1,017 lines long.  I think this is OK for a test file.